### ISSUE TITLE
results on original h36m dataset.

### ISSUE THREAD

**Issue by bucktoothsir** [JimmySuen/integral-human-pose#25]
Now I am trying to reproduce papers result(MPJPE: 40.6). I use 'integral_human_pose/experiments/hm36/resnet50v1_ft/d-mh_ps-256_deconv256x3_min-int-l1_adam_bs32-4gpus_x300-270-290/lr1e-3.yaml' and HM36_train_cache file downloaded from your dropbox. And I put h36m images in the right position.

Now i have trained for 50 epochs, but I found the MPJPE in h36m_validmin_dataset stayed around 80mm and stopped reducing.
could you please help me?

**Comment by lck1201** [comment]
Hi，50 epochs is not enough  You'd better complete 300 epochs,  and besides try other learning rates to see accuracy varying. What's more, could you post the learning curve?
